tests/unit/type/set.tcl

```diff
@@ -21,6 +21,14 @@ start_server {
     test "SADD, SCARD, SISMEMBER, SMISMEMBER, SMEMBERS basics - $type" {
         create_set myset $initelems($type)
         assert_encoding $type myset
+
+        # coverage for objectComputeSize
+        if {$type == "listpack"} {
+            assert_morethan [r memory usage myset] 58
+        } elseif {$type == "hashtable"} {
+            assert_morethan [r memory usage myset] 6080
+        }
```

Comment:
1. please add some comments explaining these constants. e.g. `;# 58 is the sum of the sizes of the elements in $initelems`
2. let's use assert_range to check some upper bound (*2 maybe).
3. i don't think we want to have a different assertion constant for each encoding, we should run the assertion twice for coverage, but the constants should be low enough to make sure the actual data is stored, and allow some overhead in the storage format.

Reply:
@oranagra, I have updated the assrtion and pushed the changes. Hope this covers comments.